.node-version

```diff
@@ -0,0 +1 @@
+16.15.1
```

Comment:
current active LTS version of node.  with node 18 installed, I ran into dependency installation issues